plugins/collections/get.js

```diff
@@ -38,15 +38,16 @@ module.exports = () => ({
                         result.type = 'normal';
                     }
 
-                    // Store promises from pipelineFactory fetch operations
-                    const collectionPipelines = [];
-
-                    result.pipelineIds.forEach(id => {
-                        collectionPipelines.push(pipelineFactory.get(id));
-                    });
+                    const listConfig = {
+                        search: {
+                            field: 'id',
+                            keyword: collection.pipelineIds
```

Comment:
should it be:
```suggestion
                            keyword: result.pipelineIds
```
?

Reply:
`result` seems to be same as `collection https://github.com/screwdriver-cd/screwdriver/pull/2272/files#diff-d851c112d50a2117dac31feeb7c49c470dfe5a1d6b95aeaacd2136d02ccdd498R31